src/main/java/uk/gov/pay/api/filter/LoggingFilter.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.pay.api.filter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+public class LoggingFilter implements Filter {
```

Comment:
This one and the associated  test can easily be in a common repository so that we don't need to copy/paste code acorss repos.